components/status_dropdown/status_dropdown.tsx

```diff
@@ -128,7 +128,7 @@ export default class StatusDropdown extends React.PureComponent<Props, State> {
     }
 
     setDnd = (index: number): void => {
-        const currentDate = getCurrentMomentForTimezone(this.props.timezone);
+        const currentDate = getCurrentMomentForTimezone(this.props.timezone as string);
```

Comment:
instead of doing this in so many places we can make the parameter optional

Reply:
and in the function check if its undefined or not then return suitably

Reply:
I think the check is already there in the function.